tests/compilation_tests/robot_test/execute_setup_test.py

```diff
@@ -1,4 +1,10 @@
-from electrical.rf_module import Device, RadioSession
```

Comment:
is it me or is everything in here all in robot_test.py? If so, why copy the code?

Reply:
I think this was outdated code -- new pushes should have cleaned this issue up